### ISSUE TITLE
Increasing evaluate_total_loss value

### ISSUE THREAD

**Issue by t13m** [athena-team/athena#308]
Is it normal for training processes on AISHELL or HKUST that the total loss value on evaluation set increases after certain steps? Specifically, for HKUST corpus, the evaluate_total_loss is about 12 at 30k step and will increase to over 15 at 100k step; for AISHELL corpus, the loss grows less, but still increases from 6 to 9. These experiments are almost conducted with default configs, only with multiple gpus (8 in total). Is this what should be expected? or is there anything wrong with hyper parameters? Maybe learning rate should be adjust accordingly when using multiple gpu? Thanks

**Comment by t13m** [comment]
evaluate_total_loss on aishell using athena (default config on 8 gpus)
<img width="903" alt="image" src="https://user-images.githubusercontent.com/5397898/95946021-49820e80-0e1e-11eb-850c-da8dd2fa9857.png">

as a comparison, evaluate loss on aishell using espnet is like this (default config on 1 gpu)
<img width="903" alt="image" src="https://user-images.githubusercontent.com/5397898/95946173-982fa880-0e1e-11eb-9e6e-c7858e2bcd28.png">

**Comment by Some-random** [comment]
@cookingbear please take a look at this

**Comment by Some-random** [comment]
Sorry for the late reply. We have observed similar trends for evaluate_total_loss and we haven't figured out why our loss curves would be different from espnet. Despite it, it wouldn't cause any degradation in performance because the attention and ctc accuracy for each epoch is almost always increasing, and that's the metrics we used when selecting checkpoints for decoding.
